.travis.yml

```diff
@@ -12,6 +12,9 @@ after_success:
 services:
   - postgresql
 
+before_install:
+- export COVERALLS_REPO_TOKEN=Y8eJAiu5n3BdD3BloaQULnqdZIywylTu3
```

Comment:
A few recommendations;

- Please look up how to run coverage and report the results in coveralls by including commands in this file.
- This token may not be required but if you find it necessary, kindly look up a way of picking it from the environment rather than hardcoding it. It is the norm to not hard code variables that are exported to an environment.

Reply:
Thank you for the recommendations. Let me get started on implementing them.

Reply:
This has been implemented.

Reply:
The implementation is not yet complete. Particularly the first recommendation.

Reply:
The application is configured to use the Django-nose test suite to run the tests which is why when the `python manage.py test` is run, the coverage is also run and displayed. The coverage report is then sent to coveralls by a call to `coveralls` in `after-success` section of the .travis.yml file

Reply:
Kindly note, that I have added this to the `background context` section of the PR.